treeherder/webapp/api/logslice.py

```diff
@@ -38,8 +39,8 @@ def list(self, request, project, jm):
         Receives a line range and job_id and returns those lines
         """
         job_id = request.QUERY_PARAMS.get("job_id")
-
-        log = jm.get_log_references(job_id)
+        log_name = request.QUERY_PARAMS.get("name", "builds-4h")
```

Comment:
I'm going to just make the api take the potential new name.  I created https://bugzilla.mozilla.org/show_bug.cgi?id=1139517 to make it store the new name.  There is a lot of test code to modify that I would rather not muddy with this PR.  But great point!  :)